build.gradle

```diff
@@ -15,7 +15,6 @@
 buildscript {
     repositories {
         maven { url 'https://repo.spring.io/plugins-release' }
-        maven { url "https://plugins.gradle.org/m2/" }
```

Comment:
I assume this was no-op?

Reply:
This is what `gradlePluginPortal()` points at anyway, I assume it was included in error at some point